src/layers/recurrent.jl

```diff
@@ -106,10 +117,10 @@ function (rnn::RNNCell{false, typeof(identity)})((x,
     return h_new, st
 end
 
-function Base.show(io::IO, r::RNNCell{bias}) where {bias}
+function Base.show(io::IO, r::RNNCell{use_bias}) where {use_bias}
     print(io, "RNNCell($(r.in_dims) => $(r.out_dims)")
     (r.activation == identity) || print(io, ", $(r.activation)")
-    bias || print(io, ", bias=false")
+    use_bias || print(io, ", use_bias=false")
```

Comment:
I think keeping this as `bias` is better. But if we are making this change, we need to do it for all the bias optional layers.

Reply:
This was actually an unintended change, I've fixed now